### PR TITLE
Bowling game menu and test

### PR DESCRIPTION
**Pull request by IscJacobo19**
# *The menu was created with a while
 # *I added test in a different file
 # *I added a file for my first class called Game.rb

### CODE REVIEW

File: bowling_game.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      bowling = [1, 4, 4, 5, 6, 4, 5, 5, 10, 0, 0, 1, 7, 3, 6, 4, 10, 0, 2, 8, 6]
+      print "#{bowling}\n"
+      score = games.balls(bowling)
+      print "#{score}\n"
+    when 4
+      print "thanks for playing \n"
+    when 5..10
+      print "Option not valid. bye\n"    
+  end
+end
```
Comment (EsmeSantoyo) [inline bowling_game.rb:40]: here
```suggestion
end

```

Comment (EsmeSantoyo) [inline bowling_game.rb]: here 
```suggestion
end

```

File: games.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class Games 
+  def balls(bowling)
+    @bowling = bowling
+  end
+end
```
Comment (EsmeSantoyo) [inline games.rb]: here
```suggestion
end

```

File: test_bowling_game.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+require 'minitest/autorun'
+require_relative 'games.rb'
+class Test_bowling_game < Minitest::Test 
+    def test_process
+      games = Games.new
+      assert_equal [30,60,90,120,150,180,210,240,270,300] ,games.balls([10, 0, 10, 0, 10, 0, 10, 0, 10, 0, 10, 0, 10, 0, 10, 0, 10, 0, 10, 10, 10])
+      assert_equal [20,37,44,74,104,134,164,194,224,254] ,games.balls([10, 0, 10, 0, 2, 5, 10, 0, 10, 0, 10, 0, 10, 0, 10, 0, 10, 0, 10, 10, 10])
+      assert_equal [5,14,29,49,60,61,77,97,117,133] ,games.balls([1, 4, 4, 5, 6, 4, 5, 5, 10, 0, 0, 1, 7, 3, 6, 4, 10, 0, 2, 8, 6])
+    end
+  end
```
Comment (EsmeSantoyo) [inline test_bowling_game.rb]: and here
```suggestion
  end
  
```

**Review by EsmeSantoyo — CHANGES_REQUESTED** [review]
You should add a final blank line and use rubocop to review your ruby syntax to avoid that kind of "errors".

File: bowling_game.rb (hunk 1 above)
Comment (IscJacobo19) [reply bowling_game.rb]: ok. thanks you!

**Comment by cfcortes** [comment]
Buen punto lo de rubocop @EsmeSantoyo lo voy a agregar en las recomendaciones

File: test_bowling_game.rb (hunk 3 above)
Comment (IscJacobo19) [reply test_bowling_game.rb]: @cfcortes  @EsmeSantoyo  Buen día, hice un comimit para corregir estos detalles, cuando se que ya está aprobado, para continuar programando?

**Review by jvargas98 — APPROVED** [review]

**Pull request merged**